web/js/components/image-download/image-download-panel.js

```diff
@@ -59,84 +72,70 @@ export default class ImageDownloadPanel extends React.Component {
       lonlats,
       { width, height },
       time,
-      fileType,
-      fileType === 'application/vnd.google-earth.kmz' ? false : isWorldfile,
+      currFileType,
+      currFileType === 'application/vnd.google-earth.kmz' ? false : currIsWorldfile,
       markerCoordinates,
     );
 
     if (url) {
```

Comment:
Is this check of `url` required? If not we should eliminate it, if so we may want to keep the `else` condition.